### ISSUE TITLE
Can't import seed phrase offline

### ISSUE THREAD

**Issue by bota87** [BlueWallet/BlueWallet#5723]
I would like to use an old android smartphone as a cold wallet so I keep it in airplane mode and set bluewallet electrum setting in offline mode but after import the seed phrase I get

> import error
> electrum client is not connected

BlueWallet version is 6.4.8

**Comment by Nftness** [comment]
Hello, have you resolve this issue?

**Comment by bota87** [comment]
I don't think that can be solved without change the code and exit with a new release

**Comment by Nftness** [comment]
exactly
